src/App.jsx

```diff
@@ -38,63 +38,68 @@ injectGlobal`
 
 class BotfuelWebChat {
   static init(param) {
-    if (!localStorage.getItem('BOTFUEL_WEBCHAT_USER_ID')) {
-      localStorage.setItem('BOTFUEL_WEBCHAT_USER_ID', uuidv4());
-    }
+    function initialize() {
+      if (!localStorage.getItem('BOTFUEL_WEBCHAT_USER_ID')) {
+        localStorage.setItem('BOTFUEL_WEBCHAT_USER_ID', uuidv4());
+      }
 
-    const updatedTheme = {
-      buttons: {
-        close: !!param.embeddedContainerId,
-      },
-      fixed: !!param.embeddedContainerId,
-    };
-    updatedTheme.buttons.close = !param.embeddedContainerId;
-    // updatedTheme.buttons.fullScreen = !param.embeddedContainerId;
-    updatedTheme.fixed = !param.embeddedContainerId;
+      const updatedTheme = {
+        buttons: {
+          close: !!param.embeddedContainerId,
+        },
+        fixed: !!param.embeddedContainerId,
+      };
+      updatedTheme.buttons.close = !param.embeddedContainerId;
+      // updatedTheme.buttons.fullScreen = !param.embeddedContainerId;
+      updatedTheme.fixed = !param.embeddedContainerId;
 
-    if (updatedTheme.fixed) {
-      document.body.innerHTML += '<div id="botfuel"></div>';
-    }
+      if (updatedTheme.fixed) {
+        document.body.innerHTML += '<div id="botfuel"></div>';
+      }
 
-    if (param.applicationId) {
-      /* eslint-disable no-console */
-      console.warn(
-        'Using applicationId in the Botfuel Webchat init script is deprecated. Please use appToken instead.',
-      );
-    }
+      if (param.applicationId) {
+        /* eslint-disable no-console */
+        console.warn(
+          'Using applicationId in the Botfuel Webchat init script is deprecated. Please use appToken instead.',
+        );
+      }
 
-    const mergedTheme = merge(defaultTheme, param.theme, updatedTheme);
+      const mergedTheme = merge(defaultTheme, param.theme, updatedTheme);
 
-    if (typeof mergedTheme.layout.rounded === 'boolean') {
-      console.log(
-        'Using a boolean as theme layout rounded property is deprecated. Please use a border-radius css string value instead.',
+      if (typeof mergedTheme.layout.rounded === 'boolean') {
+        console.log(
+          'Using a boolean as theme layout rounded property is deprecated. Please use a border-radius css string value instead.',
+        );
+        mergedTheme.layout.rounded = mergedTheme.layout.rounded ? '20px' : '15px';
+        /* eslint-enable no-console */
+      }
+      if (document.getElementById(param.embeddedContainerId || 'botfuel') === null) {
```

Comment:
this can be put in a const to be reused later:

```javascript
const container = document.getElementById(param.embeddedContainerId || 'botfuel');
```